src/app/debug/debug.component.ts

```diff
@@ -17,7 +18,7 @@ export class DebugComponent implements OnInit, AfterViewInit {
   @ViewChild(DebugTreeComponent) debugTreeComponent!: DebugTreeComponent;
   @ViewChild('splitter') splitter: any;
   @ViewChild('bottom') bottom!: ElementRef;
-  currentView: any = {};
+  currentView: CurrentTestView = {} as CurrentTestView;
```

Comment:
CurrentTestView has some required properties, now you act like those exist by type asserting with `as CurrentTestView` but this is not the proper way to do this, either make the currentView property also be able to be undefined (`currentView?: CurrentTestView`) or make all properties inside CurrentTestView optional